classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
```

Comment:
- [ ] Kindly add a getter for the `@students`

Reply:
Thank you for this feedback, I have implemented it.